code/game/gamemodes/cult/runes.dm

```diff
@@ -191,6 +191,7 @@
 		M.visible_message("<span class='warning'>[M] writhes in pain as the markings below \him glow a bloody red.</span>", \
 		"<span class='danger'>AAAAAAHHHH!.</span>", \
 		"<span class='warning'>You hear an anguished scream.</span>")
+		M.pain_level += 10
```

Comment:
Couldn't find one.
I don't know why I added this, I'm removing it even if it makes sense.